modules/unsupported/process-raster/src/test/java/org/geotools/process/raster/RasterAsPointCollectionProcessTest.java

```diff
@@ -311,6 +314,114 @@ public void testCoverageDifferentCRS()
+        GridCoverage2D coverage1 = (GridCoverage2D) processor.doOperation(param);
+        param = processor.getOperation("CoverageCrop").getParameters();
+        param.parameter("Source").setValue(myCoverage);
+        param.parameter("Envelope").setValue(new ReferencedEnvelope(-180, 180, -90, -1, crs));
+        GridCoverage2D coverage2 = (GridCoverage2D) processor.doOperation(param);
+        // Resample the coverages to LatLon coordinates
+        String wkt =
+                "GEOGCS[\"WGS 84\","
+                        + "DATUM[\"World Geodetic System 1984\","
+                        + "SPHEROID[\"WGS 84\", 6378137.0, 298.257223563, AUTHORITY[\"EPSG\",\"7030\"]],"
+                        + "AUTHORITY[\"EPSG\",\"6326\"]],"
+                        + "PRIMEM[\"Greenwich\", 0.0, AUTHORITY[\"EPSG\",\"8901\"]],"
+                        + "UNIT[\"degree\", 0.017453292519943295],"
+                        + "AXIS[\"Geodetic latitude\", NORTH],"
+                        + "AXIS[\"Geodetic longitude\", EAST],"
+                        + "AUTHORITY[\"EPSG\",\"4326\"]]";
+        crs = CRS.parseWKT(wkt);
+        param = processor.getOperation("Resample").getParameters();
+        param.parameter("Source").setValue(coverage1);
+        param.parameter("CoordinateReferenceSystem").setValue(crs);
+        coverage1 = (GridCoverage2D) processor.doOperation(param);
+        param = processor.getOperation("Resample").getParameters();
+        param.parameter("Source").setValue(coverage2);
+        param.parameter("CoordinateReferenceSystem").setValue(crs);
+        coverage2 = (GridCoverage2D) processor.doOperation(param);
+        // Execution of the RasterAsPointCollectionProcess setting hemisphere
+        SimpleFeatureCollection collection1 = process.execute(coverage1, null, null, null, true);
+        SimpleFeatureCollection collection2 = process.execute(coverage2, null, null, null, true);
+        // Check if the points are exactly as the number of pixel number
+        int pixelNumber1 =
+                coverage1.getRenderedImage().getHeight() * coverage1.getRenderedImage().getWidth();
+        Assert.assertEquals(pixelNumber1, collection1.size());
+        int pixelNumber2 =
+                coverage2.getRenderedImage().getHeight() * coverage2.getRenderedImage().getWidth();
+        Assert.assertEquals(pixelNumber2, collection2.size());
+        // Check if each point is in the correct hemisphere
+        try (SimpleFeatureIterator it = collection1.features()) {
+            while (it.hasNext()) {
+                Assert.assertEquals(NORTH, it.next().getAttribute("emisphere"));
+            }
+        }
+        try (SimpleFeatureIterator it = collection2.features()) {
+            while (it.hasNext()) {
+                Assert.assertEquals(SOUTH, it.next().getAttribute("emisphere"));
+            }
+        }
+    }
+
+    @Test
+    public void testCoverageInPolarStereographicProjections() throws Exception {
```

Comment:
Seems two tests crammed into one... could it be done with two tests, each invoking the same method with different parameters instead?

Reply:
I split the two unit tests into four tests.  The Windows build failure is not related to this PR.

Reply:
Looks good to me